Cleaning Task #1

passflaze opened the pull request:
Double check before accepting !

Added the missing methods descriptions, and updated the overall readability

passflaze committed:
Added the missing methods descriptions, and updated the overall readability

giutala reviewed (changes requested):
- add data retrieval from the library yahoo finance and ensure linearity with the rest of code. 
- consider making two classes: highfrequency_indexdata_retrieval (where you need to apply the cases of fetch twelve data and dividing data into samples ecc; and lowfrequency...

passflaze committed:
Adjusted the two functions:
Asset_helpers -> added the yahoofinance retrieval function.
df_dataretrieval ->
Added the if clause checking whether the selected timedelta (and frequency) is higher than 5k, improved overall readibility adding comments where necessary, and fixed the 'next' function to be able to skip over adjustable minutes intervals.

giutala committed:
Merge branch 'main' into riccardo_branch
